dll/win32/crtdll/CMakeLists.txt

```diff
@@ -19,12 +19,7 @@ list(APPEND SOURCE
 
 add_library(crtdll MODULE ${SOURCE})
 set_module_type(crtdll win32dll ENTRYPOINT DllMain 12)
-target_link_libraries(crtdll crt wine)
-
-if(MSVC)
-    add_importlibs(crtdll msvcrt)
-    target_link_libraries(crtdll oldnames)
-endif()
+target_link_libraries(crtdll crt wine ${PSEH_LIB})
```

Comment:
`oldnames` is not needed anymore?

Reply:
no, crtdll doesn't import anything anymore